test/test_torchscript_consistency.py

```diff
@@ -391,23 +381,32 @@ def test_Fade(self):
         fade_in_len = 3000
         fade_out_len = 3000
 
-        _test_script_module(torchaudio.transforms.Fade, waveform, fade_in_len, fade_out_len)
+        self._assert_conssitency(T.Fade(fade_in_len, fade_out_len), waveform)
 
     def test_FrequencyMasking(self):
         tensor = torch.rand((10, 2, 50, 10, 2))
-        _test_script_module(
-            torchaudio.transforms.FrequencyMasking, tensor, freq_mask_param=60, iid_masks=False)
+        self._assert_conssitency(T.FrequencyMasking(freq_mask_param=60, iid_masks=False), tensor)
 
     def test_TimeMasking(self):
         tensor = torch.rand((10, 2, 50, 10, 2))
-        _test_script_module(
-            torchaudio.transforms.TimeMasking, tensor, time_mask_param=30, iid_masks=False)
+        self._assert_conssitency(T.TimeMasking(time_mask_param=30, iid_masks=False), tensor)
 
     def test_Vol(self):
         test_filepath = os.path.join(
             common_utils.TEST_DIR_PATH, 'assets', 'steam-train-whistle-daniel_simon.wav')
         waveform, _ = torchaudio.load(test_filepath)
-        _test_script_module(torchaudio.transforms.Vol, waveform, 1.1)
+        self._assert_conssitency(T.Vol(1.1), waveform)
+
+
+class TestTransformsCPU(_TransformsTestMixin, unittest.TestCase):
+    """Test suite for Transforms module on CPU"""
+    device = torch.device('cpu')
+
+
+@unittest.skipIf(not torch.cuda.is_available(), 'CUDA not available')
+class TestTransformsCUDA(_TransformsTestMixin, unittest.TestCase):
```

Comment:
Overall, this looks reasonable to me. What would be an alternative way?

Reply:
Actually, I do not come up with an alternative that is not hacky and as good as this one.
PyTorch's test dynamically generates this by manipulating global variable scope, but that's basically same thing as this implementation., yet more messy and it does not bring any additional benefit for us.